### ISSUE TITLE
Integrating this build with stock Fedora 26 NetworkManager

### ISSUE THREAD

**Issue by izauddin** [dlenski/openconnect#69]
Hi,

Thanks for this build. Hope that it will be integrated into the Network Manager very very soon.
I'm able to build and use this build to connect to my company Palo Alto VPN / GlobalProtect. 
Currently I run it from the command line. Is there any way to integrated this with the NetworkManager / Openconnect that came with Fedora 26 ?

**Comment by dlenski** [comment]
Glad it's working!

If you know how to build NetworkManager, then see [dlenski/network-manager-openconnect](https://github.com/dlenski/network-manager-openconnect/). See comments on issue #63 for why that's trickier to build than OpenConnect alone is.

Once GlobalProtect support is merged into mainline OpenConnect, I expect a newer NM-plugin release will follow and get picked up by the major Linux distros.

**Comment by izauddin** [comment]
Thanks for the info. Look like hve to use script to automated the connection.
